index.bs

```diff
@@ -675,11 +698,9 @@ Many specifications need to be patched so that, if a given algorithm invoked in
 
 <h4 id="patch-serial">Web Serial API</h4>
 
-<div algorithm="Serial requestPort patch">
-  Modify the {{Serial/requestPort()}} method steps by inserting the following steps after the initial creation of |promise|:
+Add {{[DelayWhilePrerendering]}} to {{Serial/requestPort()}}.
 
-  1. If [=this=]'s [=relevant global object=]'s [=Window/browsing context=] is a [=prerendering browsing context=], then append the following steps to [=this=]'s [=platform object/post-prerendering activation steps list=] and return |promise|.
-</div>
+TODO: the below could probably be done by generalizing {{[DelayWhilePrerendering]}} to use owner document while in dedicated workers.
```

Comment:
I'm happy to spec this out but I wanted to give you all a chance to see the simple version first.

Reply:
Ah yep, I wrote the above before seeing this. For dedicated workers we should be able to do something like that, except shared workers can make dedicated workers.

For the initial origin trial in Chrome, we are planning to just silently defer the creation of dedicated workers until after activation.